src/common/SwipeableLoaderRow.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import * as React from 'react';
+// imported from experimental sources
+// eslint-disable-next-line
+import SwipeableRow from 'SwipeableRow';
+import LoaderRow from './LoaderRow';
+
+type Props<TEntity> = {
+  renderErrorListItem: React.ComponentType,
+  isOpen: boolean,
+  renderListItem: React.ComponentType,
+  loader: LoadObject<TEntity>,
+  renderLoadingListItem: React.ComponentType,
+  onClose: () => void,
+  onOpen: (rowKey: string) => void,
+};
+
+class SwipeableLoaderRow<TEntity> extends React.PureComponent<Props<TEntity>> {
+  _onOpen = (): void => this.props.onOpen(this.props.rowKey);
+
+  render() {
+    const {
+      renderErrorListItem,
+      renderListItem,
+      loader,
+      renderLoadingListItem,
+      renderSlideoutView,
+      ...swipeableProps
+    } = this.props;
+
+    if (!loader.hasValue()) {
+      return (
```

Comment:
this branch basically for disabling swipe for rows without value.

Reply:
super-nit: store the `LoaderRow` in a variable